flink-runtime/src/main/java/org/apache/flink/runtime/util/EnvironmentInformation.java

```diff
@@ -41,6 +41,8 @@
  * startup options, or the JVM version.
  */
 public class EnvironmentInformation {
+    public static final String UNKNOWN_COMMIT_ID = "DecafC0ffeeD0d0F00d";
+    public static final String UNKNOWN_COMMIT_ID_ABBREV = "DeadD0d0";
```

Comment:
technically speaking these should be  `@VisibleForTesting`.

Reply:
Technically you are correct - the best kind of correct. Fixed it.